src/allmydata/web/common.py

```diff
@@ -320,48 +360,10 @@ def humanize_failure(f):
 
 
 class MyExceptionHandler(appserver.DefaultExceptionHandler, object):
```

Comment:
Since `MyExceptionHandler` is a Nevow-only feature, I gutted this class and moved all of its logic to free functions that we can continue to use with Twisted Web.  I had to leave `MyExceptionHandler` in place because of `UnsupportedMethod`, though.  We have no other place to stick code to handle `UnsupportedMethod`.  Fortunately, Twisted Web handles it well so when we switch from Nevow to Twisted Web and delete `MyExceptionHandler`, `UnsupportedMethod` will continue to work as desired.